components/chainhook-sdk/src/indexer/stacks/mod.rs

```diff
@@ -502,6 +512,19 @@ pub fn standardize_stacks_block(
             signer_bitvec: block.signer_bitvec.clone(),
             signer_signature: block.signer_signature.clone(),
 
+            signer_public_keys: match (signer_sig_hash, &block.signer_signature) {
+                (Some(signer_sig_hash), Some(signatures)) => {
+                    Some(signatures.iter().map(|sig_hex| {
+                        let sig_msg = clarity::util::secp256k1::MessageSignature::from_hex(sig_hex)
+                            .expect("unable to parse signer signature message");
+                        let pubkey = get_signer_pubkey_from_message_hash(&signer_sig_hash, &sig_msg)
+                            .expect("unable to recover signer sig pubkey");
```

Comment:
Can you `map_err` instead of `expect` to propagate the error upwards / avoid a panic if this goes wrong?